### DISCUSSION TITLE
"Table d'hôte" at a restaurant generally refers to what?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#457] [Galaxy Trivia] [answered]
**Bonus:** What does "à la carte" mean?

**Comment by peterbaccaro** [comment]
In restaurant terminology, a table d'hôte is a **menu where multi-course meals with only a few choices are charged at a fixed total price**.

**Comment by thuongtruong1009** [comment, marked as answer]
"Table d'hôte" is a French term that is commonly used in restaurants to refer to a fixed-price menu where a multi-course meal is served with limited choices. This type of menu is also known as a "prix fixe" menu. The dishes on a table d'hôte menu are usually selected by the chef and change regularly based on the season and availability of ingredients. Customers typically have a choice of a few dishes for each course, but they cannot order individual items a la carte. The price of the menu usually includes all the courses of the meal, but beverages and gratuity may be charged separately. Table d'hôte menus are popular in fine dining restaurants and are often used for special occasions or events.
